.github/workflows/build-ami.yml

```diff
@@ -0,0 +1,50 @@
+name: Build AWS Neuron AMI
+
+on:
+    push:
+      branches:
+        - main
+      paths:
+        - 'aws-ami-creation/**'
+  workflow_dispatch:
+  schedule:
+    # Schedule the workflow to run every second day at midnight UTC
+    - cron: '0 0 */2 * *'
+
+jobs:
+    build-ami:
+        defaults:
+          run:
+            working-directory: aws-ami-creation
+        runs-on: ubuntu-latest
+        steps:
+
+        - uses: actions/checkout@v3
+        - uses: hashicorp/setup-packer@main
+        
+        - name: configure aws credentials
+          uses: aws-actions/configure-aws-credentials@v1
+          with:
+            aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+            aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+            aws-region: us-east-1
```

Comment:
@glegendre01 Is that okay with you? The CI runs a packer inside GitHub actions to create an AMI on AWS. Or do we use [Assume Role directly using GitHub OIDC provider](https://github.com/aws-actions/configure-aws-credentials?tab=readme-ov-file#using-this-action)

Reply:
first good point, the runner is not self-hosted, so it's ephemeral
Second point, if your ACCESS Key authorization are scoped to packer only it's fine, if it's not the case, we need to do it.
Indeed, the good solution is too use OIDC, i added on our todo list

Reply:
So its okay for now to continue until we have OIDC?

Reply:
yes

Reply:
@philschmid @glegendre01 how can we add this secrets then?

Reply:
@shub-kris, what scope (iam permissions) do you need? @glegendre01 should be able to add those

Reply:
The ones mentioned in the offical doc here: https://developer.hashicorp.com/packer/integrations/hashicorp/amazon#iam-task-or-instance-role

```
{
  "Version": "2012-10-17",
  "Statement": [
    {
      "Effect": "Allow",
      "Action": [
        "ec2:AttachVolume",
        "ec2:AuthorizeSecurityGroupIngress",
        "ec2:CopyImage",
        "ec2:CreateImage",
        "ec2:CreateKeyPair",
        "ec2:CreateSecurityGroup",
        "ec2:CreateSnapshot",
        "ec2:CreateTags",
        "ec2:CreateVolume",
        "ec2:DeleteKeyPair",
        "ec2:DeleteSecurityGroup",
        "ec2:DeleteSnapshot",
        "ec2:DeleteVolume",
        "ec2:DeregisterImage",
        "ec2:DescribeImageAttribute",
        "ec2:DescribeImages",
        "ec2:DescribeInstances",
        "ec2:DescribeInstanceStatus",
        "ec2:DescribeRegions",
        "ec2:DescribeSecurityGroups",
        "ec2:DescribeSnapshots",
        "ec2:DescribeSubnets",
        "ec2:DescribeTags",
        "ec2:DescribeVolumes",
        "ec2:DetachVolume",
        "ec2:GetPasswordData",
        "ec2:ModifyImageAttribute",
        "ec2:ModifyInstanceAttribute",
        "ec2:ModifySnapshotAttribute",
        "ec2:RegisterImage",
        "ec2:RunInstances",
        "ec2:StopInstances",
        "ec2:TerminateInstances"
      ],
      "Resource": "*"
    }
  ]
}


```

Reply:
@glegendre01